post_installation_configuration/cluster-tasks.adoc

```diff
@@ -197,9 +197,9 @@ For clusters managed by the Assisted Installer, you can add worker nodes by usin
 
 * xref:../nodes/nodes/nodes-sno-worker-nodes.adoc#sno-adding-worker-nodes-to-single-node-clusters-manually_add-workers[Manually adding worker nodes to a SNO cluster]
 
-=== Adding worker nodes to clusters managed by the multicluster engine for Kubernetes
+=== Adding worker nodes to clusters managed by the {mce}
```

Comment:
Apparently you forgot to list the preview for this assembly but I did find it https://76266--ocpdocs-pr.netlify.app/openshift-enterprise/latest/post_installation_configuration/cluster-tasks